### ISSUE TITLE
[Accessibility : Focus & sequence] Bond member save/ edit/ delete button not accessible 

### ISSUE THREAD

**Issue by atreyeemukhopadhyay** [kimchi-project/ginger#287]
in bond page when we add a new bond member or modify a existing bond member , all related buttons are not getting focused when we navigate using keyboard.After bond member dropdown focus is moving to 

![bond member button](https://cloud.githubusercontent.com/assets/13199397/14136834/9f39995a-f682-11e5-83b8-f74d21d56248.jpg)
